src/main/java/Deadline.java

```diff
@@ -0,0 +1,19 @@
+public class Deadline extends Task {
+    private final String TYPE = "D";
+    private String by;
```

Comment:
Consider declaring attributes as protected rather than private as it would be easier to facilitate extension of subclasses as the project progress.